app/controllers/external_submissions_controller.rb

```diff
@@ -7,5 +7,6 @@ class ExternalSubmissionsController < ApplicationController
   end
 
   def index
+    @models = Model.external.with_not_evaluated_hypothesis
```

Comment:
We need someone from Meetween to start the evaluation because of the user-grant relation. We are also storing logs in the meetween group, thus someone from the meetween needs to start the evaluation. Once it is started we can see the results on the main model webpage as well as in the rankings. 

In the future, maybe there will be a dedicated view for browsing external users' models.